docs/reference/release-notes/860.md

```diff
@@ -12,6 +17,78 @@ keywords:
 | ---------------------- | ---------------------------- | ------------ | ------------ |
 | 8 October 2024         | 14 April 2026                |              |              |
 
+## 8.6.0-alpha2
+
+| Release date | Changelog(s)                                                                                                                                                              |                   |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
+| 11 June 2024 | - [ Camunda 8 core ](https://github.com/camunda/zeebe/releases/tag/8.6.0-alpha2) <br /> - [ Connectors ](https://github.com/camunda/connectors/releases/tag/8.6.0-alpha2) | [Release blog](#) |
```

Comment:
```suggestion
| 11 June 2024 | - [ Camunda 8 core ](https://github.com/camunda/zeebe/releases/tag/8.6.0-alpha2) <br /> - [ Connectors ](https://github.com/camunda/connectors/releases/tag/8.6.0-alpha2.1) | [Release blog](#) |
```

Reply:
For future reference, do you usually release `alphax.0` or `alphax.1`? Or will you keep track of this with the Connectors team?

Reply:
We usually only release using the `alpha-X` pattern without any patches like everyone else. In this case we had to release a patch for the `alpha2` hence `alpha2.1` as we did not want to go with `alpha3`.